doc/Install.md

```diff
@@ -449,3 +449,58 @@ section:
 	sql_mode = '';
 
 After that, restart mysql and try again.
+
+### Your worker never or rarely runs
+
+Friendica is coded to always play nice. It checks whether the host machine is idle enough and if it _seems_ to be overloaded, it intermittently refuses to process the worker queue.
```

Comment:
please use one sentence per line for your next changes to the documentations...

Reply:
Good idea! Incidentally, I usually do that nowadays in my own docs as well. I'll fix this with the next additions. I hope to submit a few more frequently discussed ones from our chat.